### ISSUE TITLE
context menu for branch switcher to open a PR (if found)

### ISSUE THREAD

**Issue by danqing** [desktop/desktop#4114]
Glad to see that v1.1 displays PR number. However, it will be a lot more convenient if that badge is clickable, like in the old client. Now if I see my PR failing tests, I still need to open the browser and navigate to that page to see what's going on.

**Comment by shiftkey** [comment]
@danqing I'm not sure if this came up earlier (I'll dig that up later) but there's also a keyboard shortcut for this (Cmd+R on macOS or Ctrl+R on Windows) to speed it up.

**Comment by danqing** [comment]
That's good to know thanks @shiftkey ! That said, it will still be good if the UI actually supports this :)

**Comment by shiftkey** [comment]
Here's some context from the initial exploration https://github.com/desktop/desktop/issues/2642#issuecomment-330313559

> > What should happen when someone clicks on the PR badge? What about the status icon?
>
> For now, I don't think the badge should have a function. I've been going back and forth on this because it's certainly not far-fetched to imagine it would open the current PR but I'm super vary of overloading the button as well. Buttons in buttons rarely come out well. I have been exploring UI affordances for opening the current PR both in the button and in the foldout but I'm not happy with any of them at the moment and, as @j-f1 points our we will have a way (albeit not a very discoverable one) to get there.
>
> TL;DR: Initially nothing, but we'll keep exploring ways of getting quick access to the current PR.

**Comment by shiftkey** [comment]
Marking this as `more-information-needed` as it'd be nice to confirm from @desktop/design that our thinking on this hasn't changed.

**Comment by niik** [comment]
> Marking this as more-information-needed as it'd be nice to confirm from @desktop/design that our thinking on this hasn't changed.

No, our thinking hasn't change with regards to the specific suggestion to make the PR number inside of the toolbar button clickable. We have talked in the past about more versatile tooltips though which could be a bearer of actions like that as well as showing more information about the build status, similar to

<img width="601" alt="screen shot 2018-03-12 at 16 34 13" src="https://user-images.githubusercontent.com/634063/37293342-47222184-2613-11e8-82dc-bf98a4247aca.png">

I believe we're a ways off from being able to do that though.

**Comment by nerdneha** [comment]
Related to the keyboard shortcut, another roundabout way from the app is to click on the PR and use the Menu items: `Branch` > `Show Pull Request`. 

@niik @donokuda is there a possibility of adding the option to right click on a PR in the `Current Branch` dropdown to provide the `Show Pull Request` option? 

Pros:
1. don't have to click on the PR and then switch to keyboard or do another 2 clicks

Cons:
1. it's not visible, you have to know about the right click option
1. you still have to open the dropdown
1. not sure that's a significant improvement / worth the effort

FWIW, we allow right clicking on repos in the `Current Repository` dropdown to show options

**Comment by billygriffin** [comment]
Thanks for all the thoughts here. I'm going to combine this with #8152 because they're looking to solve the same problem, and we'll discuss the best solution in that issue. Thanks!
